models/inorganic_volume_filter/blur/inorganic_volume_filter_blur.sql

```diff
@@ -0,0 +1,324 @@
+{{ config(
+    alias = 'blur',
+    partition_by = ['day'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['day', 'block_time', 'nft_contract_address', 'nft_token_id', 'tx_hash', 'inorganic_filters', 'unique_trade_id'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "sector",
+                                "inorganic_volume_filter",
+                                \'["henrystats"]\') }}')
+}}
+
+{% set project_start_date = '2022-10-19' %} -- blur start date 
+
+WITH 
+
+trades as (
+        SELECT 
+            t.* 
+        FROM 
+        {{ ref('nft_trades') }} t 
+        WHERE 1 = 1
+        {% if is_incremental() %}
+        AND t.block_time >= date_trunc('day', now() - interval '1 week')
+        {% endif %}
+        AND t.project IN ('opensea', 'blur')
+        AND t.blockchain = 'ethereum'
+), 
+
+royal_settings as (
+        SELECT 
+            collection, 
+            fee
+        FROM 
+        (
+        SELECT 
+            *, 
+            ROW_NUMBER() OVER (PARTITION BY collection ORDER BY evt_block_time DESC) as ordering 
+        FROM 
+        (
+        SELECT 
+            * 
+        FROM 
+        {{ source('looksrare_ethereum', 'RoyaltyFeeRegistry_evt_RoyaltyFeeUpdate') }}
```

Comment:
Is this the right source event?

Reply:
this was the equivalent on v1 `looksrare."RoyaltyFeeRegistry_evt_RoyaltyFeeUpdate"` and I also tried querying on v2 annd it returned results, why do you think it isn't the right source event? incase I need to double check something.

Reply:
Oh it's a high level check, since we have the models split by project and both models are nearly identical otherwise, I was asking in case a copypasta error left both models referring to looksrare.

Reply:
ah okay, ty ser, it was intentional.